### ISSUE TITLE
Install on CentOS 6.7 fails

### ISSUE THREAD

**Issue by jgfoster** [GsDevKit/GsDevKit_home#156]
installOsPrereqs includes a check for `lsb_release -ds` that (at least in one case) answers `"CentOS release 6.7 (Final)"` (including the quotes). In order to get the script to recognize the OS, I had to add `cut -d '"' -f 2` to the command to remove the leading quote. After that the script got further.

**Comment by dalehenrich** [comment]
Did it get "further" enough to finish the install phase, or are you 
still working through issues?

**Comment by dalehenrich** [comment]
does `/etc/centos-release` exist in this case ... perhaps we will want to change the order of the tests?

**Comment by jgfoster** [comment]
Yes, `/etc/centos-release` exists and changing the order seems like it should work. And, no, I'm not done yet, but I'm working through issues (mostly dealing with out-of-space for both disk blocks and inodes; but also finding the right 32-bit libraries to install or make).

**Comment by dalehenrich** [comment]
Looks [like @brunobuzzi also ran into the same problem](http://forum.world.st/Installation-error-on-Centos-tt4930448.html) and has a suggested fix that seems to be the same as yours ... 

All I need is for someone to verify that this change doesn't beak other oses ... of course, when I find the time, I'll have to do that:)

**Comment by dalehenrich** [comment]
@grype could you verify that the following expression produces `Debian` on one of your Debian systems?

```bash
osName=`lsb_release -ds | cut -d ' ' -f 1 | cut -d '"' -f 2`
```

**Comment by dalehenrich** [comment]
Ah, it is confirmed for Debian ...

**Comment by dalehenrich** [comment]
fixed in PR #158
